docs/unified-test-documentation/generic-testing-stand-setup.md

```diff
@@ -0,0 +1,328 @@
+available platforms, however there are unique exceptions - for example
+[MSI boards](https://docs.dasharo.com/unified/msi/recovery/) where we use the
+JTPM headers and a FlashBIOS button. Refer to our platform-specific
+documentation when in doubt.
+
+#### Prerequisites
+
+* [Prepared RTE](../transparent-validation/rte/v1.1.0/quick-start-guide.md)
+* SOIC-8 Pomona clip (if applicable)
+* 6x female-female wire cables
+
+#### Connections
+
+If your motherboard **does not** have SPI headers available, follow this guide
+for setting up a Pomona clip connection:
+
+##### Pomona
+
+1. Connect the wire cables to the **Pomona clip**.
+
+    ![](../images/pomona_clip.jpg)
+    ![](../images/pomona_clip_with_cables.jpg)
+
+1. Connect the Pomona clip to the [SPI header](../transparent-validation/rte/v1.1.0/specification.md)
+   on RTE.
+
+    | SPI header | Pomona clip  |
+    |:----------:|:------------:|
+    | Vcc        | pin 5 (Vcc)  |
+    | GND        | pin 4 (GND)  |
+    | CS         | pin 1 (CS)   |
+    | SCLK       | pin 7 (CLK)  |
+    | MISO       | pin 2 (MISO) |
+    | MOSI       | pin 8 (MOSI) |
+
+    ![](../images/pomona_clip_with_rte.jpg)
+
+1. Open the platform cover and locate the **SPI flash chip**.
+
+    ![](../images/vp46xx_location_of_flash_chip.jpg)
+
+1. Match **pin 1(CS)** on the Pomona clip with the first pin of the flash chip,
+   marked with a small dot engraved on the chip.
+
+    ![](../images/flash_chip.jpg)
+    ![](../images/pomona_clip_connected_to_flash_chip.jpg)
+
+##### SPI headers
+
+If your platform has dedicated SPI flash headers like the Protectli VP66XX,
+consider yourself lucky and simply connect corresponding pins with Dupont
+wires:
+
+![](../images/flash_headers.jpg)
+
+| SPI header | RTE          |
+|:----------:|:------------:|
+| Vcc        | Vcc          |
+| GND        | GND          |
+| CS         | CS           |
+| SCLK       | CLK          |
+| MI**SO**   | MI**SO**     |
+| MO**SI**   | MO**SI**     |
+
+##### Remaining pins
+
+1. Locate **CMOS headers** and wire them to GPIO pins on the RTE. You usually
+need to reset the CMOS after flashing for a successful firmware update.
+![](../images/vp46xx_location_of_CMOS_header.jpg)
+    | RTE       | DUT                        |
+    |:---------:|:--------------------------:|
+    | J11 pin 8 | CLR_CMOS                   |
+    | Any GND   | GND                        |
+
+1. Locate the power and reset button pins. A motherboard will usually have a
+dedicated set of headers where the power and reset button wires are connected.
+Connect them to proper pins on the RTE. In extreme cases you might need to use
+clips to grab pins of soldered-in power and reset buttons.
+
+    | RTE       | DUT                           |
+    |:---------:|:-----------------------------:|
+    | J11 pin 5 | PWR_ON#                       |
+    | J11 pin 6 | RST#                          |
+    | J15 pin 1 | GND                           |
```

Comment:
@filipleple I have concerns regarding this table; in the case of PC Engines, pin5 is RST, and pin6 is PWR; otherwise, it doesn't work. Could that be some mistake? Can we check that for Protectli, as it seems to be a reference in this guide?